### DISCUSSION TITLE
navigation in pagination

### DISCUSSION THREAD

**Discussion by ChiMaoShuPhy** [marp-team/marp#158] [Q&A] [answered]
Hello,

I am wondering if there is a convenient way (without javascript involving) to enable navigation in pagination on every slide? so that one can quickly jump to other slides, like in [https://www.w3schools.com/w3css/w3css_pagination.asp](https://www.w3schools.com/w3css/w3css_pagination.asp)

many thanks!

**Comment by yhatt** [comment, marked as answer]
Marp slides are designed for the slide deck like a PowerPoint so we have no support for a page list that is not common in the slide deck.

However, you can set page links into the footer manually just like below:

```makdown
---
footer: >-
  [1](#1)
  [2](#2)
  [3](#3)
  [4](#4)
  [5](#5)
---

your slides...
```

Set `href` to the anchor to specific page number.

Styling example:

```html
<style>
  footer a {
    background: #fff;
    border: 1px solid #000;
    box-sizing: border-box;
    color: #000 !important;
    display: inline-block;
    height: 1.5em;
    line-height: 1;
    padding: 0.25em;
    text-align: center;
    text-decoration: none;
    width: 1.5em;
  }
</style>
```

**Reply by ChiMaoShuPhy** [reply]
Thanks a lot for the example! I guess there is no easy way if there were like a hundred of slides. :sob:

**Reply by ChiMaoShuPhy** [reply]
maybe a slide bar will do?

**Reply by yhatt** [reply]
It's time to [create a Marp (Marpit) plugin](https://marpit.marp.app/usage?id=extend-marpit-by-plugins)! Marp should be able to make a plugin for auto-assigning the footer from generated pages.
